src/services/jsTyping.ts

```diff
@@ -5,21 +5,26 @@
 
 /* @internal */
 namespace ts.JsTyping {
+    interface TsdJsonCacheInfo {
+        path: string;

```

Comment:
The path is used to update the "notFound" property.  It will also be needed to update the "lastUpdated" time stamp.